### PR TITLE
Collision & WireFrame AABB start

### PR DESCRIPTION
**Pull request by kashike**
https://github.com/AlmuraDev/AssetConverter/pull/3
https://github.com/AlmuraDev/Almura-2.0-ContentPacks/pull/31

### CODE REVIEW

File: src/main/java/com/almuradev/almura/block/builder/AbstractBlockTypeBuilder.java [hunk 1]
```diff
@@ -151,6 +176,9 @@ public BLOCK build(String id) {
         this.resistance().ifPresent(resistance -> block.setResistance((float) resistance));
         this.lightEmission().ifPresent(emission -> block.setLightLevel((float) emission));
         this.lightOpacity().ifPresent(block::setLightOpacity);
+        if (id.contains("raspberry")) {
```
Comment (kashike) [inline src/main/java/com/almuradev/almura/block/builder/AbstractBlockTypeBuilder.java]: I like raspberries, so I use them to debug. Ignore this! :dancer:

**Comment by liach** [comment]
Why not ray tracing stuff too?

**Comment by liach** [comment]
Are those formatting changes necessary?

**Comment by liach** [comment]
@kashike seems to put links to private repositories here.

**Pull request merged**